### PR TITLE
COVID19: update SEIR notebook in nb browser, workspace and chart ETL

### PR DESCRIPTION
**Pull request by paulineribeyre**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "covid19-etl": "quay.io/cdis/covid19-etl:1.0.9",
```
Comment (m0nhawk) [inline chicagoland.pandemicresponsecommons.org/manifest.json:9]:
```suggestion
    "covid19-etl": "quay.io/cdis/covid19-etl:1.0.11",
```
To match the versions, it's already was tested in qa.

Comment (paulineribeyre) [reply chicagoland.pandemicresponsecommons.org/manifest.json:9]: but it's not the latest? QA should use 1.0.11 to get the fixed data

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Pull request merged**